server/src/main/java/org/elasticsearch/index/mapper/MappingLookup.java

```diff
@@ -194,13 +180,6 @@ public NamedAnalyzer indexAnalyzer(String field, Function<String, NamedAnalyzer>
         return unmappedFieldAnalyzer.apply(field);
     }
 
-    /**
-     * Returns an iterable over all the registered field mappers (including alias mappers)
-     */
-    public Iterable<Mapper> fieldMappers() {
```

Comment:
happy that this goes away! I did not get at first that this was the main reason why you ended up working on this PR. That is to me a good reason to refactor things unless we want to remove get field mappings.